parsely_raw_data/utils.py

```diff
@@ -48,3 +49,6 @@ def get_default_parser(description, commands=None):
 def clean_network(network):
     """Format a network name to match AWS resources"""
     return network.replace(".", "-").replace(" ", "-").lower()
+
+def parse_datetime_arg(arg):
```

Comment:
Nitpick: adding docstrings that define the type of input (the type of `arg`) and output, particularly if there are no tests, can be helpful for utils functions